cime_config/SystemTests/systemtest_utils.py

```diff
@@ -39,19 +53,32 @@ def run_python_script(caseroot, this_conda_env, command, tool_path):
                 command, shell=True, check=True, text=True, stdout=f, stderr=subprocess.STDOUT
             )
     except subprocess.CalledProcessError as error:
-        print("ERROR while getting the conda environment and/or ")
-        print(f"running the {tool_name} tool: ")
-        print(f"(1) If your {this_conda_env} environment is out of date or you ")
-        print(f"have not created the {this_conda_env} environment, yet, you may ")
-        print("get past this error by running ./py_env_create ")
-        print("in your ctsm directory and trying this test again. ")
-        print("(2) If conda is not available, install and load conda, ")
-        print("run ./py_env_create, and then try this test again. ")
-        print("(3) If (1) and (2) are not the issue, then you may be ")
-        print(f"getting an error within {tool_name} itself. ")
-        print("Default error message: ")
-        print(error.output)
-        raise
+        # First, retry with the original method
```

Comment:
In. this context I don't know what "original method" means. But, I think you are saying run with "conda activate" rather than "conda run". So it tries with "conda run" and if that fails, it tries again with "conda activate".  Being more explicit in the comment about that's what it's doing would be helpful here.

Reply:
Good point; I improved comments in d1ed672.